packages/ag-charts-community/src/scale/colorScale.test.ts

```diff
@@ -64,3 +64,81 @@ test('multi-value domain', () => {
     expect(scale.convert(500)).toBe('#ffffff');
     expect(scale.convert(501)).toBe('#ffffff');
 });
+
+test('heatmap multistop', () => {
```

Comment:
Nit: Could you add an enclosing `describe()` block for this entire module whilst you're here?